convert-stage-4.sh

```diff
@@ -202,41 +148,55 @@ install_files() {
       ;;
   esac
 
-  sudo install -d ${primary_dir}/${identitydir}
-  sudo install -d ${primary_dir}/${inventorydir}
-  sudo install -d ${primary_dir}/${sysconfdir}
-  sudo install -d ${primary_dir}/${sysconfdir}/scripts
-
   sudo ln -sf /data/${dataconfdir} ${primary_dir}/${localstatedir}
 
-  sudo install -m 0755 ${mender_client} ${primary_dir}/${bindir}/mender
-
-  sudo install -t ${primary_dir}/${identitydir} -m 0755 \
-      ${mender_dir}/mender-device-identity
+  # Call mender make install target
```

Comment:
Maybe you should check if golang is available, mender is downloaded and GOPATH set? It is all done in the docker conversion, but what if one runs the `mender-convert` simply from the shell?

Reply:
Good point. I will add the checks.